airbyte-api/src/main/openapi/config.yaml

```diff
@@ -4103,46 +4102,47 @@ components:
       description: "Describes the difference between two Streams."
       required:
         - transformType
+        - fieldName
       properties:
         transformType:
           type: string
           enum:
             - add_field
             - remove_field
             - update_field_schema
+        fieldName:
+          $ref: "#/components/schemas/FieldName"
         addField:
-          $ref: "#/components/schemas/FieldNameAndSchema"
+          $ref: "#/components/schemas/FieldAdd"
```

Comment:
why separate schemas for FieldAdd and FieldRemove? is there a benefit to this?

Reply:
@alovew I could be convinced this was the wrong thing. What I want to make clear in the API is the pattern that each transformation type gets its own, separately named object. For add / remove it is slightly redundant. But when we add something like "add is hash-able" or something that it is clear that should go in its own field with its own object. 

Basically was trying to make the pattern clear and thought the redundancy of one object was worth it. wdyt?

Reply:
Ah I see, yes this makes sense to me. I'm good with this 👍